packages/psnotify.vm/tools/chocolateyinstall.ps1

```diff
@@ -10,38 +10,31 @@ try {
 
     $unzipLocation = 'C:\'  # psnotify has a requirement of being located in 'C:\psnotify'
     $toolDir = Join-Path $unzipLocation $toolName
-    try {
-        $toolDir = Join-Path $unzipLocation $toolName
-        $shortcutDir = Join-Path ${Env:TOOL_LIST_DIR} $category
+    $shortcutDir = Join-Path ${Env:TOOL_LIST_DIR} $category
 
-        # Remove files from previous zips for upgrade
-        VM-Remove-PreviousZipPackage ${Env:chocolateyPackageFolder}
+    # Remove files from previous zips for upgrade
+    VM-Remove-PreviousZipPackage ${Env:chocolateyPackageFolder}
 
-        # Download and unzip
-        $packageArgs = @{
-            packageName    = ${Env:ChocolateyPackageName}
-            unzipLocation  = $unzipLocation
-            url            = $zipUrl
-            checksum       = $zipSha256
-            checksumType   = 'sha256'
-            url64bit       = $zipUrl_64
-            checksum64     = $zipSha256_64
-        }
-        Install-ChocolateyZipPackage @packageArgs
-        VM-Assert-Path $toolDir
-        $executablePath = Join-Path $toolDir "$toolName.exe" -Resolve
-        $shortcut = Join-Path $shortcutDir "$toolName.lnk"
-
-        $executableDir  = $toolDir
-        Install-ChocolateyShortcut -shortcutFilePath $shortcut -targetPath $executablePath -WorkingDirectory $executableDir
-        VM-Assert-Path $shortcut
-
-        Install-BinFile -Name $toolName -Path $executablePath
-        return $executablePath
-    } catch {
-        VM-Write-Log-Exception $_
+    # Download and unzip
+    $packageArgs = @{
+        packageName    = ${Env:ChocolateyPackageName}
+        unzipLocation  = $unzipLocation
+        url            = $zipUrl
+        checksum       = $zipSha256
+        checksumType   = 'sha256'
+        url64bit       = $zipUrl_64
+        checksum64     = $zipSha256_64
     }
+    Install-ChocolateyZipPackage @packageArgs
     VM-Assert-Path $toolDir
+    $executablePath = Join-Path $toolDir "$toolName.exe" -Resolve
+    $shortcut = Join-Path $shortcutDir "$toolName.lnk"
+
+    $executableDir  = $toolDir
+    Install-ChocolateyShortcut -shortcutFilePath $shortcut -targetPath $executablePath -WorkingDirectory $executableDir
+    VM-Assert-Path $shortcut
+
+    return $executablePath
```

Comment:
This return should be removed. It was in the helper to use from the calling code (outside the helper).